sw/device/lib/dif/dif_plic.h

```diff
@@ -0,0 +1,271 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _SW_DEVICE_LIB_DIF_PLIC_H_
+#define _SW_DEVICE_LIB_DIF_PLIC_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+/**
+ * PLIC IRQ IDs enumeration
+ *
+ * Enumeration of all PLIC interrupt source IDs.
```

Comment:
These comments can be merged into one line. This applies to the other enums in this file.